src/lib/protocols/dns.c

```diff
@@ -315,6 +315,28 @@ static int search_valid_dns(struct ndpi_detection_module_struct *ndpi_struct,
 		  )) {
 		memcpy(&flow->protos.dns.rsp_addr, packet->payload + x, data_len);
 	      }
+        else if (rsp_type == 0x0c)
```

Comment:
Indentation seems to be off for this section

Reply:
Spaces vs tabs ill make sure to use tabs going forward seems to be the preferred indenting